src/main/java/com/bisnode/opa/client/rest/UrlNormalizer.java

```diff
@@ -0,0 +1,23 @@
+package com.bisnode.opa.client.rest;
+
+import java.util.Optional;
+
+class UrlNormalizer {
```

Comment:
I'd rather not make "utilities". How about creating `OpaUrl` class that would hold the original url and have methods like `asSanitizedUrl`/`asNormalizedUrl`?

Reply:
Cool idea